examples/raysgd_nyctaxi.py

```diff
@@ -84,42 +93,66 @@ def setup(self, config):
             models=[nyc_model], optimizers=[optimizer], criterion=criterion)
         self.model = self.model[0]
         self.optimizer = self.optimizer[0]
-        # Get the corresponging shard
-        train_shard = train_dataset.get_shard(self.world_rank)
-        train_loader = DataLoader(train_shard, batch_size=64)
-        test_shard = test_dataset.get_shard(self.world_rank)
-        val_loader = DataLoader(test_shard, batch_size=64)
-        self.register_data(train_loader=train_loader, validation_loader=val_loader)
-
-# You can either train the model like this
-
-trainer = TorchTrainer(training_operator_cls=CustomOperator,
-                       num_workers=num_executors,
-                       add_dist_sampler=False,
-                       num_cpus_per_worker=1,
-                       config={"lr":0.01})
-for i in range(10):
-    stats = trainer.train()
-    print(stats)
-    val_stats = trainer.validate()
-    print(val_stats)
+        self.register_data(train_loader=train_dataset, validation_loader=test_dataset)
+
+class PrintingCallback(TrainingCallback):
+    def handle_result(self, results: List[Dict], **info):
+        print(results)
+
+def train_func(config):
+    device = torch.device(f"cuda:{train.local_rank()}" if
+                          torch.cuda.is_available() else "cpu")
+    if torch.cuda.is_available():
+        torch.cuda.set_device(device)
+
+    # Set the args to whatever values you want.
+	# If using DistributedSampler, modify it here.
+    training_operator = CustomOperator(
```

Comment:
With ray train, we no longer need this operator, right? Let's use pure ray train(which is the new version of raysgd), check their [examples](https://docs.ray.io/en/latest/train/examples.html).